test/easyblocks/general.py

```diff
@@ -98,17 +99,15 @@ def test_custom_easyblocks_repo(self):
         """Test whether using a custom easyblocks repo works as expected."""
         easyblocks_path = up(os.path.abspath(__file__), 3)
 
-        # determine path to where easybuild.framework/vsc is imported from, so we can hard set $PYTHONPATH
+        # determine path to where easybuild.framework is imported from, so we can hard set $PYTHONPATH
         # this is required to dance around issues with easy-install.pth files determining the actual Python search path
         # see also http://blog.olgabotvinnik.com/blog/2014/03/03/2014-03-03-pythonpath-is-a-liar-site-py-and-easy-install-pth-tell/
         import easybuild.framework
         framework_path = up(easybuild.framework.__file__, 3)
-        import vsc.utils.generaloption
-        vsc_path = up(vsc.utils.generaloption.__file__, 3)
 
         # prepend path to easybuild-easyblocks repo to $PYTHONPATH, so we're in full(?) control
         pythonpaths = os.environ.get('PYTHONPATH', '').split(os.pathsep)
-        os.environ['PYTHONPATH'] = os.pathsep.join([easyblocks_path, framework_path, vsc_path])
+        os.environ['PYTHONPATH'] = os.pathsep.join([easyblocks_path, framework_path] + pythonpaths)
```

Comment:
Wait, how did this work before? `pythonpaths` was not used?

Reply:
I'm also not sure I understand why the split/join on pathsep is needed?

Reply:
It worked fine before despite the bug of basically ignoring the current `$PYTHONPATH` because if you had easyblocks, framework and `vsc-base` in `$PYTHONPATH`, that was fine.

I guess there's little point in doing the `split`, but it doesn't hurt either?

Reply:
Doesn't it make things needlessly complex?